README.md

```diff
@@ -1,30 +1,19 @@
 [![Documentation Status](https://readthedocs.org/projects/aiidalab-widgets-base/badge/?version=latest)](https://aiidalab-widgets-base.readthedocs.io/en/latest/?badge=latest)
 
-# Base widgets for AiiDAlab
+# AiiDAlab Widgets
```

Comment:
I am open to renaming the package, but obviously this **must** be discussed and agreed upon within the team.